src/frontend/display/btmodelviewreaddisplay.cpp

```diff
@@ -207,6 +208,24 @@ void BtModelViewReadDisplay::print(TextPart const type,
     }
 }
 
+void BtModelViewReadDisplay::speakSelectedText() {
+    if (!m_textToSpeech)
+        m_textToSpeech.reset(new QTextToSpeech);
+
+    // use language from module for TTS
+    const int selectedColumnIndex = m_parentWindow->getSelectedColumn();
+    assert(m_parentWindow->modules().size() > selectedColumnIndex);
```

Comment:
Please use `BT_ASSERT` instead of regular `assert`.

Reply:
Done.